.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+name: git2go CI
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+    - v*
+
+jobs:
+
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        go: [ '1.9', '1.10', '1.11', '1.12' ]
```

```suggestion
        go: [ '1.9', '1.10', '1.11', '1.12' , '1.13']
```

aww man, so much time has passed since this PR was opened.